Editor/EditorCore/EditorUtility.cs

```diff
@@ -250,11 +250,20 @@ internal static bool HasStaticFlag(this GameObject go, StaticEditorFlags flags)
         }
 
         /// <summary>
-        /// Move a GameObject to the active scene, where active scene may be a prefab stage.
+        /// Move a GameObject to the proper active root.
+        /// Checks if a default parent exists, otherwise it adds the object as a root of the active scene, which can be a prefab stage.
         /// </summary>
         /// <param name="gameObject"></param>
-        internal static void MoveToActiveScene(GameObject gameObject)
+        internal static void MoveToActiveRoot(GameObject gameObject)
         {
+#if UNITY_2021_1_OR_NEWER
```

Comment:
Problem was raised in 2020.2.0b3, is this `SceneView.GetDefaultParentObjectIfSet` only available from 2021?